tests/scripts/save_policy_to_safetensors.py

```diff
@@ -75,15 +75,16 @@ def get_policy_stats(env_name, policy_name, extra_overrides):
     # HACK: We reload a batch with no delta_timestamps as `select_action` won't expect a timestamps dimension
     dataset.delta_timestamps = None
     batch = next(iter(dataloader))
-    obs = {
-        k: batch[k]
-        for k in batch
-        if k in ["observation.image", "observation.images.top", "observation.state"]
-    }
-
-    actions_queue = (
-        cfg.policy.n_action_steps if "n_action_steps" in cfg.policy else cfg.policy.n_action_repeats
-    )
+    obs = {}
+    for k in batch:
+        if "observation" in k:
```

Comment:
nit: align more precisely with our implicit system for data key naming

```suggestion
        if k.startswith("observation"):
```